api/src/main/java/io/strimzi/api/kafka/model/listener/LoadBalancerListenerConfiguration.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.api.kafka.model.listener;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import io.sundr.builder.annotations.Buildable;
+import lombok.EqualsAndHashCode;
+
+/**
+ * Configures LoadBalancer listeners
+ */
+@JsonPropertyOrder({"serverKey"})
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@Buildable(
+    editableEnabled = false,
+    generateBuilderPackage = false,
+    builderPackage = "io.fabric8.kubernetes.api.builder"
+)
+@EqualsAndHashCode(callSuper = true)
+public class LoadBalancerListenerConfiguration extends KafkaListenerExternalConfiguration {
```

Comment:
Do we really need these empty classes? Can't we just use `KafkaListenerExternalConfiguration` when there is no other configuration? (just asking, not sure about the difference)

Reply:
Yeah, 3 of 4 ext. listeners have just empty class, but `io.strimzi.api.kafka.model.listener.IngressListenerConfiguration` has some other properties, so I did it in this way (I am not saying it is the best one). I am open to all ideas so if you wish me to rewrite it, I will.

Reply:
@tombentley WDYT?

Reply:
I agree it's pointless adding a bunch of empty classes. And since we don't seem to be using subclasses, there don't seem to be adding anything.